langchain4j-voyage/src/main/java/dev/langchain4j/model/voyage/RequestLoggingInterceptor.java

```diff
@@ -0,0 +1,82 @@
+package dev.langchain4j.model.voyage;
+
+import okhttp3.Headers;
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+import okio.Buffer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import static java.util.stream.StreamSupport.stream;
+
+public class RequestLoggingInterceptor implements Interceptor {
+
+    private static final Logger log = LoggerFactory.getLogger(RequestLoggingInterceptor.class);
+
+    private static final Pattern BEARER_PATTERN = Pattern.compile("(Bearer\\s)(\\w{2})(\\w+)(\\w{2})");
```

Comment:
major: I can see my API key logged in clear text

Reply:
Sorry about that I didn't double check the regular expression. Already fixed!